src/Services/Masa.Dcc.Service/Services/OpenApiService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc;
```

Comment:
Why you need mvc?

Reply:
[FromBody] Attribute and so on is in  the namespace